### DISCUSSION TITLE
Cert Junior Dev Users & Roles: Question 3: What can be defined via permissions out of the box in Pimcore Admin Interface?

### DISCUSSION THREAD

**Discussion by iprokhorov** [pimcore/pimcore#17932] [Q&A / Support] [answered]
Question 3: What can be defined via permissions out of the box in Pimcore Admin Interface?
- Field based permissions in data objects.
- Permissions to most of main menu entries.
- Access rules for Pimcore elements via workspaces.
- Visible Custom Layouts and Perspectives.
- Access rules based on property values of elements.

Somehow, I don’t quite understand what I’m supposed to answer here. The answers confuse me—I’ve already answered all the  questions for every cert, but it’s all stuck on this one, and I just don’t get what exactly is being asked. Is it about the general concept of permissions or some specific functionality (e.g. The "Permissions" section in the "Settings" tab of the user/role)? 

Or is this question about workspaces too?

Whatever I try, it’s wrong. I’m starting to lose patience.

**Comment by iprokhorov** [comment, marked as answer]
thank you for your great support ....

**Comment by Gintron** [comment]
Hey I am struggling with this too, I am afraid I don`t understand the question

**Reply by iprokhorov** [reply]
I'm not sure anymore, but I think I figured it out:

- Permissions to most of main menu entries.
- Access rules for Pimcore elements via workspaces.
- Visible Custom Layouts and Perspectives.




That's really not my style to just leak the answer here (if it's even correct :-D), but I think the question is really oddly phrased, and there's just no support available.
